src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -165,7 +165,7 @@ public static IndexReader getReader(String path) throws IOException {
    * @return stemmed form of the term or <code>null</code> if error encountered
    */
   public static String analyzeTerm(String term) {
-    return analyzeTerm(term, DEFAULT_ANALYZER);
+    return analyzeTermWithAnalyzer(term, DEFAULT_ANALYZER);
```

Comment:
Why rename? Overloading seems fine?

Reply:
It works fine in Java but Pyjnius can't seem to distinguish between the two based on signature and throws an error.

Reply:
Ah. 👍

Reply:
To document the issue:
https://forum.image.sc/t/minimum-working-example-for-pyjnius-method-overload-issues/11329
https://github.com/kivy/pyjnius/issues/427